### PR TITLE
3P Media: Category can expand and collapse

### LINKED ISSUE #2360: Make category container expandable / collapsable

### PR DESCRIPTION
**Pull request by ezxyz**
## Summary

Adds ability to collapse and expand the categories. 

## To-do

Add percy tests #3358 

## User-facing changes

2dc3076e-1632-45f9-bbce-e0be5a5c25b0

## Testing Instructions

Fixes #2360

### CODE REVIEW

File: assets/src/edit-story/components/library/panes/media/media3p/media3pCategories.js [hunk 1]
```diff
@@ -18,34 +18,78 @@
...
-      // TODO(#2362) Wire up pill list to state and remove these fake pills.
-      fakePillList.map((e, i) => (
-        <CategoryPill isSelected={e == selectedPill} key={i} title={e} />
-      ))
-    }
-  </CategorySection>
-);
+  return (
+    <CategorySection>
+      <CategoryPillContainer isExpanded={isExpanded}>
```
Comment (swissspidy) [inline assets/src/edit-story/components/library/panes/media/media3p/media3pCategories.js]: I am missing `aria-expanded` here

File: assets/src/edit-story/components/library/panes/media/media3p/media3pCategories.js [hunk 2]
```diff
@@ -18,34 +18,78 @@
...
+        {
+          // TODO(#2362) Wire up pill list to state and remove these fake pills.
+          fakePillList.map((e, i) => (
+            <CategoryPill
+              isSelected={i == selectedPill}
```
Comment (swissspidy) [inline assets/src/edit-story/components/library/panes/media/media3p/media3pCategories.js:77]: Can multiple pills be selected? If so, `aria-pressed` might be suitable here.

If not, `CategoryPillContainer` should have `role="tablist"`, and the pills should have `role="tab"`. See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role

Let's not make accessibility be an after thought 🙂

File: assets/src/edit-story/components/library/panes/media/media3p/media3pCategories.js (hunk 1 above)
Comment (ezxyz) [reply assets/src/edit-story/components/library/panes/media/media3p/media3pCategories.js]: Added.

File: assets/src/edit-story/components/library/panes/media/media3p/media3pCategories.js (hunk 2 above)
Comment (ezxyz) [reply assets/src/edit-story/components/library/panes/media/media3p/media3pCategories.js:77]: 1) I added role="tablist" to the CategoryPillContainer. 

2) I added role="tab" and aria-selected={props.isSelected} to the CategoryPill component in categoryPill.js.

File: assets/src/edit-story/components/button/index.js [hunk 3]
```diff
@@ -30,6 +30,7 @@ import {
   Redo as RedoIcon,
   ArrowLeft as ArrowLeftIcon,
   ArrowRight as ArrowRightIcon,
+  ArrowDown,
```
Comment (cvolzke4) [inline assets/src/edit-story/components/button/index.js]: [nit]: Should this be aliased as well, since the others also are? For consistency.

**Review by cvolzke4 — COMMENTED** [review]
Looks great. Later we may consider adding an animation.

**Review by cvolzke4 — APPROVED** [review]

File: assets/src/edit-story/components/button/index.js (hunk 3 above)
Comment (ezxyz) [reply assets/src/edit-story/components/button/index.js]: Done.

**Pull request merged**